features/google/src/components/importExplanationExpandable/importExplanationExpandable.css

```diff
@@ -0,0 +1,157 @@
+    letter-spacing: -0.01em;
+    margin: 0 0 16px 0;
+}
+
+.import-view .explanation-expandable .head {
+    width: 100%;
+    max-width: var(--max-width);
+    height: 96px;
+    border-top: 1px solid var(--color-gray-400);
+    overflow: hidden;
+    display: flex;
+    align-items: center;
+    padding: 24px;
+    box-sizing: border-box;
+}
+
+.import-view .explanation-expandable .head .number {
+    width: 48px;
+    height: 48px;
+    border-radius: 50%;
+    font-size: 28px;
+    font-weight: 600;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: var(--color-indigo-950);
+    color: var(--text-light);
+}
+
+.import-view .explanation-expandable .head .heading {
+    font-family: "Jost Medium";
+    margin-left: 18px;
+    font-size: 24px;
+    flex: 1 1 auto;
+}
+
+.import-view .explanation-expandable .head .rotate-180 {
+    transform: rotate(180deg);
+}
+
+.import-view .explanation-expandable .section-body {
+    padding: 16px 16px 16px 16px;
+    width: 100%;
+    max-width: var(--max-width);
+    box-sizing: border-box;
+    border-top: 1px solid var(--color-gray-400);
+}
+
+.import-view .explanation-expandable .section-body .full-screen {
+    width: 100%;
+    max-width: var(--max-width);
+    margin-bottom: 25px;
+}
+
+.import-view .explanation-expandable .section-body .file-info p {
+    font-size: 16px;
+    color: var(--color-indigo-950);
+    margin: 0 0 2px 0;
+    line-height: 120%;
+}
+
+.import-view .explanation-expandable .section-body .x-divider {
+    display: flex;
+    flex-direction: row;
+    margin: 0;
+}
+
+.import-view .explanation-expandable .separator {
+    height: 1px;
+    width: 100%;
+    max-width: var(--max-width);
+    background-color: var(--color-gray-400);
+    margin-bottom: 26px;
+}
+
+.import-view .explanation-expandable .translucent-sides {
+    mask-image: linear-gradient(
+        to top,
+        transparent,
+        transparent,
+        rgba(0, 0, 0, 1),
+        transparent,
+        transparent
+    );
+    -webkit-mask-image: linear-gradient(
+        to top,
+        transparent,
+        transparent,
+        rgba(0, 0, 0, 1),
+        transparent,
+        transparent
+    );
+}
+
+.import-view .explanation-expandable .section-body * {
+    margin-bottom: 16px;
+}
+
+.import-view .explanation-expandable .section-body p {
+    margin-top: 0;
+    margin-bottom: 36px;
+}
+
+.import-view .explanation-expandable .section-body {
+    line-height: 120%;
+    letter-spacing: -0.01em;
+    font-size: 20px;
+}
+
+.import-view .explanation-expandable .section-body h4,
+h5 {
+    color: var(--color-indigo-950);
+    margin-top: 0;
+}
+
+.import-view .explanation-expandable .section-body h4 {
+    font-family: "Jost Medium";
+    font-size: 24px;
+    margin-bottom: 36px;
+    margin-top: 20px;
+}
+
+.import-view .explanation-expandable .section-body h5 {
+    font-weight: 500;
+    font-size: 20px;
+    margin-top: 10px;
```

Comment:
4-pixel grid...

Reply:
I don't understand very well what you mean, sorry.

Reply:
Sorry, don't margins use also a 4 pixel grid? 10px would not follow that.

Reply:
Ok, now I understand what you mean 😊 

These are not the real CSS styles. This is just a copy of the importer screen from the **Facebook import** in order to make the **google importer screen** look better, but it is provisional. (I understand that margins should follow the 4 pixels rule also in the Facebook importer, not just here, but that is not the purpose of this PR and was already approved by the designers).